subprojects/language-groovy/src/testFixtures/groovy/org/gradle/testing/fixture/GroovyCoverage.groovy

```diff
@@ -99,28 +101,37 @@ class GroovyCoverage {
         !GroovySystem.version.endsWith("-SNAPSHOT")
     }
 
-    private static Set<String> versionsAbove(Collection<String> versionsToFilter, String threshold) {
+    private static Set<String> versionsNotLessThan(Collection<String> versionsToFilter, String threshold) {
```

Comment:
`versionsNotLessThan` is a little more difficult to reason about compared to something like `versionsAtLeast` or `versionsGreaterThanOrEqual`. Thoughts?